docs/gaia/delegator-guide-cli.md

```diff
@@ -0,0 +1,388 @@
+```
+
+If you run your own full-node, just use `tcp://localhost:26657` as the address. 
+
+Then, let us set the default value of the `--trust-node` flag:
+
+```bash
+gaiacli config trust-node false
+
+// Set to true if you run a light-client node, false otherwise
+```
+
+Finally, let us set the `chain-id` of the blockchain we want to interract with:
+
+```bash
+gaiacli config chain-id gos-3
+```
+
+## Querying the state
+
+`gaiacli` lets you query all relevant information from the blockchain, like account balances, amount of bonded tokens, outsanding rewards, governance proposals and more. Next is a list of the most useful commands for delegator. Please make sure you [set up gaiacli](#setting-up-gaiacli) before trying them.
+
+```bash
+// query account balances and other account-related information
+gaiacli query account
+
+// query the list of validators
+gaiacli query validators
+
+// query the information of a validator given their address (e.g. cosmos10snjt8dmpr5my0h76xj48ty80uzwhraqalu4eg)
+gaiacli query validator <validatorAddress>
+
+// query all delegations made from a delegator given their address (e.g. cosmos10snjt8dmpr5my0h76xj48ty80uzwhraqalu4eg)
+gaiacli query delegations <delegatorAddress>
+
+// query a specific delegation made from a delegator to a validator given their addresses
+gaiacli query delegations <delegatorAddress> <validatorAddress>
+
+// query the rewards of a delegator given a delegator address (e.g. cosmos10snjt8dmpr5my0h76xj48ty80uzwhraqalu4eg)
+gaiacli query distr rewards <delegatorAddress> 
+
+// query all proposals currently open for depositing
+gaiacli query proposals --status deposit_period
+
+// query all proposals currently open for voting
+gaiacli query proposals --status voting_period
+
+// query a proposal given its proposalID
+gaiacli query proposal <proposalID>
+```
+
+For more commands, just type:
+
+```bash
+gaiacli query
+```
+
+For each command, you can use the `--h` flag to get more information.
+
+## Bonding Atoms and Withdrawing rewards
+
+::: warning
+**Before bonding Atoms, please read the [delegator faq](https://cosmos.network/resources/delegators) to understand the risk and responsabilities involved with delegating**
+:::
+
+::: warning
+**Note: These commands need to be run on an online computer. It is more secure to perform them commands using a ledger device. For the offline procedure, click [here](#signing-transactions-from-an-offline-computer).**
+::: 
+
+```bash
+// Bond Atoms 
+// ex value for flags: <amountToBound>=10000stake, <bech32AddressOfValidator>=cosmosvaloper18thamkhnj9wz8pa4nhnp9rldprgant57pk2m8s, <gasPrice>=0.001stake
+
+gaiacli tx staking --amount <amountToBond> --validator <bech32AddressOfValidator> --from <delegatorKeyName> --gas auto --gas-prices <gasPrice>
+
+// Withdraw rewards
+
+gaiacli tx distr withdraw-rewards --from <delegatorKeyName>
+```
+
+To confirm that your transaction went through, you can use the following queries:
```

Comment:
Maybe also mention `gaiacli query tx` here using the tx hash returned by creation of the transaction.